app/src/main/java/org/feature/fox/coffee_counter/util/worker/BalanceWorker.kt

```diff
@@ -11,6 +11,15 @@ import kotlinx.coroutines.withContext
 import org.feature.fox.coffee_counter.data.repository.UserRepository
 import org.feature.fox.coffee_counter.di.services.AppPreference
 
+/**
+ * TODO Add description.
+ *
+ * @constructor
+ * TODO
+ *
+ * @param context
+ * @param workerParams
+ */
```

Comment:
I think this worker could be removed too, since we not using it.

Reply:
Since I didn't work on this: Are you sure about that? Because I think that the MainActivity is using that BalanceWorker.

Reply:
Yes, but it does not do anything useful yet and can not be used as intended. I have removed it with commit 6574e4da3a106933078d43d9f80720a00c386469